game/script-topics.rpy

```diff
@@ -4779,6 +4799,265 @@ label talk_mod_contributions:
+            n 1fcsajl "You're already proving that well enough.{w=0.5}{nw}"
+            extend 1fllunl " I-{w=0.1}I think."
+
+        else:
+            n 1fcsajl "I-{w=0.1}I guess that at {i}least{/i} means you have good taste.{w=0.5}{nw}"
+            extend 1fllunl " I suppose that counts for something."
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.LOVE:
+        n 1klrss "But yeah,{w=0.1} so..."
+
+    elif jn_affinity.get_affinity_state() >= jn_affinity.ENAMORED:
+        n 1ksrss "A-{w=0.1}anyway..."
+
+    elif jn_affinity.get_affinity_state() >= jn_affinity.HAPPY: 
+        n 1flrun "A-{w=0.1}anyway."
+
+    else:
+        n 1flrun "A-{w=0.1}anyway!{w=0.5}{nw}" 
+        extend 1fcsaj " That's beside the point!"
+
+    n 1kslsr "..."
+    n 1ullaj "I guess what I'm trying to say is I still have all these memories of {i}that{/i} guy..."
+    n 1nsrpu "And although he obviously wasn't you,{w=0.5}{nw}"
+    extend 1tsraj " you kinda have {i}his{/i} memories too?{w=0.5}{nw}"
+    extend 1tslem " And..."
+    n 1fcsaj "...and..."
+    n 1fcsan "..."
+    n 1fcsem "Rrrgh,{w=0.1}{w=0.5}{nw}" 
+    extend 1fllem " this is so confusing!"
+    n 1fcsem "Ugh...{w=0.5}{nw}"
+    extend 1nnmpo " you know what?"
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.AFFECTIONATE:
+        n 1nllss "It doesn't really matter at this point,{w=0.1} does it?"
+
+    else:
+        n 1fllbo "I'm just gonna start over.{w=0.5}{nw}"
+        extend 1unmaj " Mentally,{w=0.1} I mean."
+
+    n 1ncsaj "He was here {i}then{/i}."
+    n 1fcssm "You are here {i}now{/i}."
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.AFFECTIONATE:
+        n 1fchbg "And that's all there is to it."
+
+        if jn_affinity.get_affinity_state() >= jn_affinity.LOVE:
+            if not name_match:
+                extend 1fchsm " Yep."
+                n 1uchsml "Love you,{w=0.1} [mc_initial]-"
+                n 1fllbgl "I mean,{w=0.5}{nw}"
+                extend 1kchbgl " {i}[player]~{/i}."
+                n 1fsqsml "..."
+                $ chosen_tease = random.choice(jn_globals.DEFAULT_PLAYER_TEASE_NAMES)
+                n 1uchbsl "Oh,{w=0.1} lighten up,{w=0.1} [chosen_tease]!"
+                n 1fwrtsl "You should know I'd never mean it.{w=0.5}{nw}"
+                extend  " Ehehe."
+
+            else:
+                n 1uchsml "Love you,{w=0.1} [player]~.{w=0.5}{nw}"
+                extend 1nchsml " Ehehe."
+
+    else:
+        n 1fllss "I-{w=0.1}I just gotta adjust,{w=0.5}{nw}" 
+        extend 1fllun " that's all."
+
+    return
+
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_realizations_other_girls",
+            unlocked=True,
+            prompt="Monika and the other girls",
+            conditional="""jn_utils.get_total_gameplay_length().total_seconds() / 3600 >= 12 
+                and get_topic('talk_realizations_player_ddlc_actions').shown_count > 0""",
```

Comment:
Handy thing about strings in python, we can make this look a lil cleaner all in all via
```suggestion
            conditional=(
                "jn_utils.get_total_gameplay_length().total_seconds() / 3600 >= 12 "
                "and get_topic('talk_realizations_player_ddlc_actions').shown_count > 0"
            ),
```